app/src/test/java/com/github/multimatum_team/multimatum/util/MockFirebaseAuth.kt

```diff
@@ -0,0 +1,8 @@
+package com.github.multimatum_team.multimatum.util
+
+import com.google.firebase.auth.FirebaseAuth
+import org.mockito.Mockito
+
+class MockFirebaseAuth {
+    val auth: FirebaseAuth = Mockito.mock(FirebaseAuth::class.java)
+}
```

Comment:
Shouldn't this be an object?

Reply:
For now yes, there is no particular reason for `MockFirebaseAuth` to be a class, but it will definitely need to be a class once I figure out how to mock it properly (by that I mean reproduce Firebase behavior as I did for `MockFirebaseFirestore`)